### PR TITLE
EkaterinaTRenameFolderTest

### PR DESCRIPTION
**Pull request by KaterinaTerekhova**
added positive test by TC_010_003

### CODE REVIEW

**Review by korotushak96 — COMMENTED** [review]
When we use Asser.assertEquals(), Actual result (what is on the page) should be the first parameter and Expected result (what we provide) should be second.

File: src/test/java/RenameFolderTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class RenameFolderTest extends BaseTest {
+
+    @Test
+    public void testTC_010_003_RenameFolder(){
+        final String newName = "Schedule2022";
+
+        WebDriver driver = getDriver();
```
Comment (korotushak96) [inline src/test/java/RenameFolderTest.java:13]: why do you create this variable if you don't use it?

Comment (KaterinaTerekhova) [reply src/test/java/RenameFolderTest.java:13]: yes, I just forgot to delete it.Thank you

Comment (KaterinaTerekhova) [reply src/test/java/RenameFolderTest.java:13]: and thank you for Assert reminder

**Pull request closed without merging**